backend/app/models/user.rb

```diff
@@ -36,4 +36,27 @@ class User < ApplicationRecord
   delegate :admin?, to: :role
   delegate :teacher?, to: :role
   delegate :student?, to: :role
+
+  after_create :slack_notify
+
+  private
+
+  def slack_notify
+    SlackService.call(message)
+  end
+
+  def message
```

Comment:
`message` is also a non specific name for a method like this, I rather go with something like `after_creation_message`